docs/get-started/getting-started-workload-qualification.md

```diff
@@ -0,0 +1,141 @@
+After that we will know which workload is more suitable to GPU, which is called "workload qualification" here.
+
+If there are certain operators which can not run on GPU due to the current limitations, they will fallback to CPU mode.
+As a result, it may incur some performance overhead because of host memory <=> GPU memory transfer.
+However it is not critical to ensure all operations are supported by the GPU and significant performance benefits can 
+be gained before all operations are fully supported by the GPU. It all depends on how critical the portion that is 
+executing on the CPU is to the overall performance of the query.
+
+This article is to help you get familiar with the best practice and different tools we provide on how to do gap analysis 
+and workload qualification.
+
+## 1. Qualification and Profiling tool
+
+### Requirement
+
+- Access to Spark event logs from Spark 2.x or 3.x
+- Spark 3.0.1+ jars
+- `rapids-4-spark-tools` jar
+
+### How to use
+
+If you have Spark event logs from prior runs of the applications on Spark 2.x or 3.x, you can use the
+[Qualification tool](../spark-qualification-tool.md) and [Profiling tool](../spark-profiling-tool.md) to analyze them.
+The qualification tool outputs the score, rank and some of the potentially not-supported features for each Spark application.
+Its output can help you focus on the top N Spark applications which are SQL heavy applications. 
+
+The profiling tool outputs SQL plan metrics and also prints out actual query plans to provide more insights.
+The following example profiling tool output for a specific Spark application shows that it has a query with a large 
+`HashAggregate` and `SortMergeJoin`. Those are indicators for a good candidate for the RAPIDS Accelerator.
+
+```
++--------+-----+------+----------------------------------------------------+-------------+------------------------------------+-------------+----------+
+|appIndex|sqlID|nodeID|nodeName                                            |accumulatorId|name                                |max_value    |metricType|
++--------+-----+------+----------------------------------------------------+-------------+------------------------------------+-------------+----------+
+|1       |88   |8     |SortMergeJoin                                       |11111        |number of output rows               |500000000    |sum       |
+|1       |88   |9     |HashAggregate                                       |22222        |number of output rows               |600000000    |sum       |
+```
+
+Since the two tools are only analyzing Spark event logs, it can not provide very accurate gap analysis information.
+However it is very convenient because you do not need a CPU or GPU Spark cluster to run them.
+
+## 2. Function `explainPotentialGPUPlan` 
+
+### Requirement
+
+- A Spark 3.x CPU cluster
+- A pair of `rapids-4-spark` and `cudf` jars
+
```

Comment:
maybe we should put modify your code to call up here

Reply:
Added:
```
- Need to modify the existing Spark application code

```